INSTALL_DOCKER.rst

```diff
@@ -32,15 +32,15 @@ If your user does not already have ``sudo`` rights, you should add your user to
 
   ::
 
-    usermod -G wheel -a <USERNAME>
+    $ usermod -G wheel -a <USERNAME>
```

Comment:
+1 on @gorozco1's comment on choice of prompt though. We should use `$` for non-priv commands and `#` for priv ones.